src/apps.json

```diff
@@ -2990,6 +2990,14 @@
         "Podcastly"
       ]
     },
+    {
+      "name": "Podcatcher",
+      "pattern": "^Podcatcher ",
```

Comment:
This is a very generic name, let's narrow down the pattern as much as possible.  How about: `"pattern": "^Podcatcher \\d",`